File: documentation/user/en/solve/examples/routing/get-by-unique-attribute.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeEquals("code", "macbook-pro-13-2022")
        	),
        	Require(
        		EntityFetch(
        			AttributeContentAll()
        		)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/routing/get-by-unique-attribute.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeEquals("code", "macbook-pro-13-2022")
				),
				require(
					entityFetch(
						attributeContentAll()
					)
				)
			)
		);
	}
);